app/services/api/auth/login.js

```diff
@@ -1,43 +1,37 @@
 angular.module('login.service', [])
     .factory('loginService',
-    ['$q', '$timeout', '$http', 'dashboardService',
-        function ($q, $timeout, $http, dashboardService) {
+    ['$q', 'credentials',
+        function ($q, credentials) {
```

Comment:
@wasims-cuelogic there should be function defined in the file and send name while defining the factory. For example 

    angular
        .module('login.service', [])
        .factory('loginService', [loginService]);

    function loginService() {

Also why we have credentials dependancy other there and where we are using it?